### DISCUSSION TITLE
Vite dev server doesn't log success after an error is resolved

### DISCUSSION THREAD

**Discussion by jdheeter** [quasarframework/quasar#12982] [General - Components / Directives / etc] [answered]
During development while running the Vite Dev server it's normal for the dev server to pick up some invalid code and throw an error like `vite] Internal server error: Transform failed with 1 error:`

The problem is that after the error is resolved, there is no feedback in the dev server log. When using webpack it's very obvious when the dev server has refreshed and is now serving correctly or still stuck on an error.

While the Vite dev server is working correctly, it's not always obvious from a developers perspective, because the errors are not followed by any success messages in the logs.

**Comment by rstoenescu** [comment, marked as answer]
https://github.com/quasarframework/quasar/issues/13013 will be available in q/app-vite beta.12
